src/tests/functional/inference_engine/transformations/sr_sub_graph_ops.cpp

```diff
@@ -8,18 +8,33 @@
 #include <ngraph/opsets/opset5.hpp>
 #include <cpp/ie_cnn_network.h>
 
+#include <ngraph/pass/manager.hpp>
+#include "graph_comparator.hpp"
+
 using namespace ngraph;
 
+namespace {
+
+    void init_unique_names(std::shared_ptr<ngraph::Function> f, const std::shared_ptr<ngraph::pass::UniqueNamesHolder>& unh) {
```

Comment:
Please move all this function to the ngraph_test_utils.hpp

Reply:
done